rosbag2_py/src/rosbag2_py/_transport.cpp

```diff
@@ -240,14 +247,18 @@ class Player
   }
 
   void play_impl(
-    const rosbag2_storage::StorageOptions & storage_options,
+    const std::vector<rosbag2_storage::StorageOptions> & storage_options,
     PlayOptions & play_options,
     bool burst = false,
     size_t burst_num_messages = 0)
   {
     install_signal_handlers();
     try {
-      auto reader = rosbag2_transport::ReaderWriterFactory::make_reader(storage_options);
+      std::vector<rosbag2_transport::Player::reader_storage_options_pair_t> readers_with_options{};
+      for (const auto & options : storage_options) {
```

Comment:
reserve space for vector
```suggestion
      std::vector<rosbag2_transport::Player::reader_storage_options_pair_t> readers_with_options{};
      readers_with_options.reserve(storage_options.size());
      for (const auto & options : storage_options) {
```

Reply:
changed in 82814f011d9f938889bc50d4d9060b95b1de0689